app/controllers/admin/users_controller.rb

```diff
@@ -8,6 +8,12 @@ def index
     @users = @search.result
   end
   
+  def show
+    @total_questions = Question.count
+    @total_topics = Topic.count
+    @users = User.all    
```

Comment:
@CHILL99 - show action requires user id
and shows information about user
the basic idea that you shouldn't use show fro general app information